connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -2373,10 +2344,46 @@ export async function upsertDatabaseStructuredDataFromCache({
     // We overwrite the whole table since we just fetched all child pages.
     truncate: true,
   });
-  await dbModel.update({ structuredDataUpsertedTs: new Date() });
+  // Same as above, but without the `dustId` column
+  const csvForDocument = await renderDatabaseFromPages({
+    databaseTitle: null,
+    pagesProperties,
+    cellSeparator: ",",
+    rowBoundary: "",
+  });
+  const parents = await getParents(
+    connector.id,
+    databaseId,
+    new Set<string>(),
+    runTimestamp.toString()
+  );
+  localLogger.info("Upserting Notion Database as Document.");
+  await upsertToDatasource({
+    dataSourceConfig,
+    documentId: `notion-database-${databaseId}`,
+    documentContent: {
+      prefix: `${databaseName}\ncsvHeader`,
```

Comment:
csvHeader?

Reply:
So this is the header right ? What you don't want is DB title ?

happy to remove csv header from the body though

Reply:
Ok I see it ! Fixed (removed DB name from prefix, did we want it ?)

Reply:
used the renderPrefixSection with the defaults -- should I override to have more than 64 tokens for the header ?

Reply:
Ok ended up doubling it.